Open Judge System/Workers/OJS.Workers.ExecutionStrategies/JavaSpringAndHibernateProjectExecutionStrategy.cs

```diff
@@ -37,15 +37,14 @@ public class JavaSpringAndHibernateProjectExecutionStrategy : JavaProjectTestsEx
         public JavaSpringAndHibernateProjectExecutionStrategy(
             string javaExecutablePath,
             Func<CompilerType, string> getCompilerPathFunc,
-            string javaLibsPath,
+            string javaLibrariesPath,
             string mavenPath)
-            : base(javaExecutablePath, getCompilerPathFunc, javaLibsPath)
+            : base(javaExecutablePath, getCompilerPathFunc, javaLibrariesPath)
         {
             this.MavenPath = mavenPath;
-            this.ClassPath = $"-cp {this.JavaLibsPath}*;{this.WorkingDirectory}\\target\\* ";
         }
 
-        // GroupId - > ArtifactId,Version
+        // GroupId - > ArtifactId, Version
```

Comment:
???

Reply:
Simply an annotation for what the grouping of the collection is, as Tuples are a little bit obfuscating Maven packages have ArtifactId and Version as their primary attribute properties.

Reply:
Ok, but I did had to think a while until I found out what you mean. Maybe it should be explained a little better. In a bit more descriptive way.